src/res/translation/translation_pl_PL.ts

```diff
@@ -118,7 +118,7 @@
     <message>
         <location filename="../../aboutdlgbase.ui" line="72"/>
         <source>Copyright (C) 2005-2021 The Jamulus Development Team</source>
-        <translation type="unfinished"></translation>
+        <translation>Copyright (C) 2005-2021 Zespół programistów Jamulus-a</translation>
```

Comment:
We should wait with the merge depending on what we settle on in https://github.com/jamulussoftware/jamulus/pull/1097#discussion_r581265793

Reply:
I think we can just leave it like that. If the qm file isn't something we need to worry about I'd say this is ready to be merged then.